include/laser_filters/intensity_bias_filter.h

```diff
@@ -0,0 +1,26 @@
+#ifndef INTENSITY_BIAS_FILTER_H
+#define INTENSITY_BIAS_FILTER_H
+
+#include "filters/filter_base.h"
+#include "sensor_msgs/LaserScan.h"
+
+namespace laser_filters
+{
+
+class IntensityBiasFilter : public filters::FilterBase<sensor_msgs::LaserScan>
+{
+  public:
+    IntensityBiasFilter();
+    bool configure();
+
+    bool update(const sensor_msgs::LaserScan& scan_in, sensor_msgs::LaserScan& scan_out);
+
+  private:
+    int num_total_beams_;
+    int high_intensity_threshold_;
```

Comment:
You should probably use a float here to match the type of data in the LaserScan parameter